cfme/tests/containers/test_containers_properties.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl as list_tbl_pod
+from cfme.containers.route import Route, list_tbl as list_tbl_route
+from cfme.containers.project import Project, list_tbl as list_tbl_proj
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP-9911 # CMP-9877 # CMP-9867
+
+
```

Comment:
well doing that will get you the list via mgmtsystem which will be outside of CFME. Presumably you want the list of pods associated with the provider from inside CFME, ie "what CFME says". In this case you probably want to go to the details page and click on one of the relationship links. The summary attribute of the provider should help you out here.